layouts/_shortcodes/customer-success/sku-tables.html

```
@@ -0,0 +1,44 @@
+{{- $categories := dict }}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)